### PR TITLE
Adds information regarding linking models on the hub to a HF Space

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: docs/hub/spaces-config-reference.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ Specify the version of the selected SDK (Streamlit or Gradio).
 All versions of Gradio are supported.  
 All versions of Streamlit from `0.79.0` are supported.
 
+**`models`**: _string_
+List of all the model identifiers associated with the space (for e.g. `reach-vb/musicgen-large-fp16-endpoint`).
```
Comment (julien-c) [inline docs/hub/spaces-config-reference.md:32]: it's already a few lines below

File: docs/hub/spaces-overview.md [hunk 2]
```diff
@@ -131,3 +131,19 @@ Select "Clone repository", and then you'll be able to follow the instructions to
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone2.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone1.png"/>
 </div>
+
+## Linking Models on the Hub
+
+You can showcase all the models that your space links to by adding their model identifier in your Space's README. To do so you can define them under the `models` key. For example:
```
Comment (osanseviero) [inline docs/hub/spaces-overview.md:137]:
```suggestion
You can showcase all the models and datasets that your Space links to by adding their identifier in your Space's README metadata. To do so, you can define them under the `models` and `datasets` keys. For example:
```

**Review by osanseviero — COMMENTED** [review]
Thanks!

File: docs/hub/spaces-config-reference.md (hunk 1 above)
Comment (julien-c) [reply docs/hub/spaces-config-reference.md:32]: so you can remove

**Comment by Vaibhavs10** [comment]
@osanseviero - lmk if this is okay for you.

File: docs/hub/spaces-overview.md [hunk 3]
```diff
@@ -131,3 +131,19 @@ Select "Clone repository", and then you'll be able to follow the instructions to
...
+title: My lovely space
+emoji: 🤗
+colorFrom: blue
+colorTo: green
+sdk: docker
+pinned: false
+models:
+- reach-vb/musicgen-large-fp16-endpoint
+```
+
```
Comment (osanseviero) [inline docs/hub/spaces-overview.md:149]: Would you mind also adding something along the lines of the second point in https://huggingface.slack.com/archives/C02EMARJ65P/p1693213871117809?thread_ts=1693042670.654759&cid=C02EMARJ65P please?

Comment (Vaibhavs10) [reply docs/hub/spaces-overview.md:152]: Good point! Done!

File: docs/hub/spaces-overview.md [hunk 4]
```diff
@@ -131,3 +131,22 @@ Select "Clone repository", and then you'll be able to follow the instructions to
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone2.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone1.png"/>
 </div>
+
+## Linking Models on the Hub
+
+You can showcase all the models and datasets that your Space links to by adding their identifier in your Space's README metadata. To do so, you can define them under the `models` and `datasets` keys. In addition to listing the aretefacts in the README file, you can also record them in any `.py`, `.ini` or `.html` file as well. We'll parse it auto-magically!
```
Comment (julien-c) [inline docs/hub/spaces-overview.md:137]:
```suggestion
You can showcase all the models and datasets that your Space links to by adding their identifier in your Space's README metadata. To do so, you can define them under the `models` and `datasets` keys. In addition to listing the artefacts in the README file, you can also record them in any `.py`, `.ini` or `.html` file as well. We'll parse it auto-magically! 
```

File: docs/hub/spaces-overview.md [hunk 5]
```diff
@@ -131,3 +131,22 @@ Select "Clone repository", and then you'll be able to follow the instructions to
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone2.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone1.png"/>
 </div>
+
+## Linking Models on the Hub
```
Comment (julien-c) [inline docs/hub/spaces-overview.md:135]:
```suggestion
## Linking Models and datasets on the Hub
```

File: docs/hub/spaces-overview.md [hunk 6]
```diff
@@ -131,3 +131,22 @@ Select "Clone repository", and then you'll be able to follow the instructions to
...
+
+You can showcase all the models and datasets that your Space links to by adding their identifier in your Space's README metadata. To do so, you can define them under the `models` and `datasets` keys. In addition to listing the aretefacts in the README file, you can also record them in any `.py`, `.ini` or `.html` file as well. We'll parse it auto-magically! 
+
+Here's an example linking two models with a space:
```
Comment (julien-c) [inline docs/hub/spaces-overview.md:139]:
```suggestion
Here's an example linking two models from a space:
```

File: docs/hub/spaces-overview.md [hunk 7]
```diff
@@ -131,3 +131,22 @@ Select "Clone repository", and then you'll be able to follow the instructions to
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone2.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/SSHClone1.png"/>
 </div>
+
+## Linking Models and datasets on the Hub
```
Comment (osanseviero) [inline docs/hub/spaces-overview.md:135]:
```suggestion
## Linking Models and Datasets on the Hub
```

**Review by osanseviero — APPROVED** [review]
Thanks a lot :fire:

**Pull request merged**